src/client/containers/ProfilePage/ProfilePage.styles.css

```diff
@@ -61,7 +61,7 @@ li.card-profile-result-list-item {
 }
 
 h2.card-profile-result-title {
-  font-size: 20px;
+  font-size: 25px;
```

Comment:
Try to use `rem` instead of pixels for the font-size

Reply:
ok Daniel. Thank you.